pagopa/io-app
# chore(IT Wallet): [SIW-433] Update PID issuing flow with new design system components

### This PR depends on #5197 

## Short description
This PR updates the PID issuing flow with new DS components.

## How to test
Start it-wallet with DS toggle enabled. Try Discovery snd Issuing flow. The new DS components should be used.
